src/assets/scss/abstracts/_helpers.scss

```diff
@@ -0,0 +1,6 @@
+.hidden {
+    visibility: hidden;
+  }
+  .visible {
+    visibility: visible;
+  }
```

Comment:
This part is not well formatted. It is a good practice to automate that in the IDE to do it on save or use a pre-commit hook.

Reply:
I normally use eslint - but that only lints javascript and if I set up autoformat on save I get a conflict with eslint.
Any solution you can recommend ? Maybe another linter dedicated to only CSS?